src/context/src/asset_manager.rs

```diff
@@ -176,6 +176,50 @@ impl AssetManager {
             None => None,
         }
     }
+    /// Create and return a color buffer
+    pub fn get_color_buffer(&mut self, (width, height): (u16, u16)) -> Option<ColorBuffer> {
+        match self.factory_impl {
+            FactoryImpl::OpenGL {
+                ref mut factory,
+            } => {
+                let color_buffer = Box::new(amethyst_renderer::target::ColorBuffer::new(factory, (width, height)));
+                let color_buffer_impl = ColorBufferImpl::OpenGL {
+                    color_buffer: color_buffer,
+                };
+                let color_buffer = ColorBuffer {
+                    color_buffer_impl: color_buffer_impl,
+                };
+                Some(color_buffer)
+            },
+            #[cfg(windows)]
+            FactoryImpl::Direct3D => {
+                unimplemented!();
+            },
+            FactoryImpl::Null => None,
+        }
+    }
+    /// Create and return a geometry buffer
+    pub fn get_geometry_buffer(&mut self, (width, height): (u16, u16)) -> Option<GeometryBuffer> {
```

Comment:
I know this stuff was here before this PR, but... what's the difference between `GeometryBuffer` and `ColorBuffer`?